stdlib/contextlib.pyi

```diff
@@ -75,6 +75,12 @@ class _GeneratorContextManager(AbstractContextManager[_T_co, bool | None], Conte
             self, type: type[BaseException] | None, value: BaseException | None, traceback: TracebackType | None
         ) -> bool | None: ...
 
+@overload
+def contextmanager(func: Callable[_P, Generator[_T_co]]) -> Callable[_P, _GeneratorContextManager[_T_co]]: ...
```

Comment:
Perhaps the type here should be `Generator[_T_co, None, object]` instead? `contextmanager` does not actually care what the return type of the generator is. E.g. this:

```python
@contextmanager
def f():
    yield
    return 1
```

is fine (though pointless).